src/main/java/com/gtnewhorizons/neid/core/NEIDCore.java

```diff
@@ -44,11 +44,15 @@ public String getSetupClass() {
 
     @Override
     public void injectData(Map<String, Object> data) {
-        NEIDTransformer.isObfuscated = !((boolean) Launch.blackboard.get("fml.deobfuscatedEnvironment"));
+        isObfuscated = (boolean) data.get("runtimeDeobfuscationEnabled");
```

Comment:
Doesnt this need to be inverted?

Reply:
nah, this is correct.
`fml.deobfuscatedEnvironment` is true when in a deobfuscated Environment
`runtimeDeobfuscationEnabled` is true when in an obfuscated environment, so it activates the deobfuscation process

Reply:
Oh, didnt notice the key changed